avro-builder/src/main/java/com/linkedin/avroutil1/builder/operations/codegen/own/AvroUtilCodeGenOp.java

```diff
@@ -103,6 +104,20 @@ public void run(OperationContext opContext) throws Exception {
 
   }
 
+  private void parseAvscFiles(Set<File> avscFiles, boolean areFilesImportable, AvroParseContext context,
```

Comment:
javadoc - what is parsedFiles? (i assume files previously parsed during the same "run" which are importable to files being parsed now)